src/librustc_trans/back/link.rs

```diff
@@ -77,7 +77,7 @@ pub fn get_linker(sess: &Session) -> (PathBuf, Command, Vec<(OsString, OsString)
         Command::new(linker)
     };
 
-    if let Some(ref linker) = sess.opts.cg.linker {
+    if let Some(ref linker) = sess.opts.debugging_opts.linker {
```

Comment:
I think you've changed the wrong variable...

```
[00:18:57] error[E0609]: no field `linker` on type `rustc::session::config::DebuggingOptions`
[00:18:57]   --> librustc_trans/back/link.rs:80:56
[00:18:57]    |
[00:18:57] 80 |     if let Some(ref linker) = sess.opts.debugging_opts.linker {
[00:18:57]    |                                                        ^^^^^^ unknown field
[00:18:57]    |
[00:18:57]    = note: available fields are: `codegen_backend`, `verbose`, `span_free_formats`, `identify_regions`, `emit_end_regions` ... and 89 others
```

Reply:
er oops!